infrastructure/bin/set_php_versions.sh

```diff
@@ -10,10 +10,9 @@ set -o nounset
 # Assign PHP version from arg if supplied. Runs default PHP bin without.
 PHP_VERSION=$1
 
-sudo update-alternatives --set php /usr/bin/php${PHP_VERSION}
-sudo update-alternatives --set phar /usr/bin/phar${PHP_VERSION}
-sudo update-alternatives --set phpdbg /usr/bin/phpdbg${PHP_VERSION}
-sudo update-alternatives --set php-cgi /usr/bin/php-cgi${PHP_VERSION}
-sudo update-alternatives --set phar.phar /usr/bin/phar.phar${PHP_VERSION}
+# GitHub runner only includes one version of PHP that is not necesarrily the value of the assigned therefore the Personal Package Archive (PPA) is necessary.
+LC_ALL=C.UTF-8 sudo add-apt-repository ppa:ondrej/php
+sudo apt update
```

Comment:
I'm getting a warning that `apt does not have a stable CLI interface`
```suggestion
sudo apt-get update
```

Reply:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/2c77bf2c-1ef7-4085-98f0-b75cce463d52)